src/store/includes-labels.ts

```diff
@@ -1,10 +1,16 @@
 /**
- * Checks if DOM has any out-of-stock related text.
+ * Checks if DOM has any related text.
  *
  * @param domText Complete DOM of website.
- * @param oosLabels Out-of-stock labels.
+ * @param searchLabels Search labels for a match.
  */
-export function includesLabels(domText: string, oosLabels: string[]): boolean {
+export function includesLabels(domText: string, searchLabels: string[]): boolean {
 	const domTextLowerCase = domText.toLowerCase();
-	return oosLabels.some(label => domTextLowerCase.includes(label));
```

Comment:
I think this was fine the way it was written. Any reason for the change?

Reply:
Agreed, it was fine the way it was.  It was a remainder of another update that got away from me.

Reply:
I reverted it back to using `array.some()`. Thanks for the feedback

Reply:
Ahh, no problem. Just curious if we goofed something up. No biggie. Thanks!